packages/netsuite-adapter/src/client/sdf_client.ts

```diff
@@ -457,6 +473,16 @@ export default class SdfClient {
       executor,
       timeoutInMinutes
     )
+    const importResult = actionResult.data as ImportObjectsResult
+    const failedTypeToInstances = new collections.map.DefaultMap<string, string[]>(() => [])
+
+    importResult.failedImports.forEach(failedImport => {
+      if (failedImport.customObject.result.message.includes('unexpected error')) {
+        log.debug('Failed to fetch (%s) instance due to SDF unexpected error: (%s)', failedImport.customObject.type, failedImport.customObject.id)
```

Comment:
nit: It kinda looks like the second `%s` should contain a message about the error
maybe: 
```suggestion
        log.debug('Failed to fetch (%s) instance with id (%s) due to SDF unexpected error', failedImport.customObject.type, failedImport.customObject.id)
```
Also, do you think there is value in adding `failedImport.customObject.result.message` to the log?

Reply:
I don't think that there is any reason to log it. We filter only a specific error and this error anyway is printed as part of SDF output.